todo-app/src/TodoList.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import TodoItem from './TodoItem';
+import {observer} from 'mobx-react';
+import {todoStore} from './stores';
+
+@observer
```

Comment:
Don't forget: in the recent version of `mobx-react` you need to also add `@inject`.

Reply:
Added :)